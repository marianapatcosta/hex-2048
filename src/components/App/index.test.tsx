import React from 'react'
import { render } from '@testing-library/react'
import { App } from './'

describe('App', () => {
  test('should render without errors and match snapshot', () => {
    const {
      container: { firstChild },
    } = render(<App />)

    expect(firstChild).toMatchSnapshot()
  })


  test('should render `Game Board` component', () => {
    const { getByTestId } = render(<App />)
    expect(getByTestId('game-board')).toBeInTheDocument()
  })
})
