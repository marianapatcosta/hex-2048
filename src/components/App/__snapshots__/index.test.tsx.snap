// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without errors and match snapshot 1`] = `
<header>
  <h2>
    Hex 
    <img
      alt="hex 2048 logo"
      src="/hex-2048-logo.svg"
    />
  </h2>
</header>
`;
